www/docs/errors.md

```diff
@@ -77,6 +77,18 @@ The provider you tried to use failed when setting [PKCE or Proof Key for Code Ex
 The `code_verifier` is saved in a cookie called (by default) `__Secure-next-auth.pkce.code_verifier` which expires after 15 minutes.
 Check if `cookies.pkceCodeVerifier` is configured correctly. The default `code_challenge_method` is `"S256"`. This is currently not configurable to `"plain"`, as it is not recommended, and in most cases it is only supported for backward compatibility.
 
+#### ERR_JOSE_ALG_NOT_WHITELISTED
```

Comment:
I cannot seem to find a reference to this error code in our source code. Are you sure it is originated from `next-auth`?

Reply:
ok, that's embarrassing - correct fix, but I was generating custom JWT in the same method and got the error messages crossed. Actual one is ERR_JWK_KEY_SUPPORT and https://next-auth.js.org/errors#jwt_session_error 

[next-auth][error][jwt_session_error] 
https://next-auth.js.org/errors#jwt_session_error JWKKeySupport: the key does not support HS512 verify algorithm
    at check (/Users/melissa/Documents/GitHub/busyness/busyness/node_modules/jose/lib/jwa/index.js:45:11)
    at check (/Users/melissa/Documents/GitHub/busyness/busyness/node_modules/jose/lib/jwa/index.js:58:12)
    at jwsVerify (/Users/melissa/Documents/GitHub/busyness/busyness/node_modules/jose/lib/jws/verify.js:150:5)
    at Object.module.exports [as verify] (/Users/melissa/Documents/GitHub/busyness/busyness/node_modules/jose/lib/jwt/verify.js:235:5)
    at Object.<anonymous> (/Users/melissa/Documents/GitHub/busyness/busyness/node_modules/next-auth/dist/lib/jwt.js:105:30)
    at Generator.next (<anonymous>)
    at asyncGeneratorStep (/Users/melissa/Documents/GitHub/busyness/busyness/node_modules/next-auth/dist/lib/jwt.js:22:103)
    at _next (/Users/melissa/Documents/GitHub/busyness/busyness/node_modules/next-auth/dist/lib/jwt.js:24:194)
    at /Users/melissa/Documents/GitHub/busyness/busyness/node_modules/next-auth/dist/lib/jwt.js:24:364
    at new Promise (<anonymous>) {
  code: 'ERR_JWK_KEY_SUPPORT'
}

Reply:
so since the error is originated from jwt_session_error, it should be put under there, I think